src/bin/deku_node.ml

```diff
@@ -69,12 +69,27 @@ let handle_block_by_hash =
 
 (* POST /block-level *)
 (* Retrieve height of the chain? *)
+
```

Comment:
I guess this empty line should be removed.
By the way, would you mind delete the `?` on the previous line?